### DISCUSSION TITLE
docs: server error when trying to access Date Picker docs

### DISCUSSION THREAD

**Discussion by paul-nallet** [chakra-ui/ark#2872] [General]
### Description

When i visit this link https://ark-ui.com/react/docs/components/date-picker i've got an error that block me
Application error: a server-side exception

### Link to Reproduction (or Detailed Explanation)

https://ark-ui.com/react/docs/components/date-picker

### Steps to Reproduce

go to https://ark-ui.com/react/docs/components/date-picker

### Ark UI Version

doc

### Framework

- [X] React
- [X] Solid
- [X] Vue

### Browser

chrome / firefox

**Comment by Omikorin** [comment]
Fixed by #2869

Keeping issue open to investigate this later

Thank you!

**Comment by segunadebayo** [comment]
Closing — docs site has been updated and is accessible.
